Feature/TA-7-Success-Page

PeshoOReda opened the pull request:
- Renamed `AppImages.successScreen` to `AppImages.successLogo` for clarity.
- Updated `AppRoutes` to use `successRoute` instead of `successScreen`.
- Modified the layout of the `SuccessScreen` to use `Center` for better alignment.
- Adjusted the size and fit of the success logo image.
- Updated text styles and alignment for "Thank You!!" and "The order delivered successfully" messages.
- Added line breaks to translation strings for better formatting in Arabic and English.

Signed-off-by: PeshoOReda <drmafiaemail13@gmail.com>

<!--📌 Pull Request Template -->

<!--📝 Description -->

This template ensures that all pull requests adhere to the project's contribution standards. It enforces proper naming conventions, code quality, CI/CD validation, and testing requirements. Following this checklist helps maintain a clean and efficient codebase while ensuring compatibility with JIRA issue tracking.

## 📌 Pull Request Checklist

### ✅ General
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] The PR title follows the required format: `type/TA-<issue-number>-TaskName`.
- [x] The branch name follows the required format: `feature/meaningful-name`, `fix/issue-description`, etc.

### 📂 Code Quality
- [x] All filenames are in `snake_case`.
- [x] All functions and variable names follow Dart naming conventions.
- [x] All new code is formatted with `dart format .`

### 🛠 CI/CD Validation
- [x] Linting passes with `dart analyze`.
- [x] The branch name is validated against the allowed patterns.
- [x] The PR title is validated against the required format.

### 🚀 Testing
- [ ] Unit tests are included for all new features and bug fixes.
- [ ] `flutter test --coverage` has been run and the results reviewed.
- [ ] The project builds successfully without errors.

### 📝 Additional Information
<!-- Provide any extra context or details about the changes in this PR. -->

### 🔗 Related JIRA Ticket(s):
- [x] JIRA Issue: `TA-<issue-number>`

### 📸 Screenshots (if applicable):
<!-- Add screenshots to show the changes visually. -->
![Screenshot_20250510_181703](https://github.com/user-attachments/assets/24ff7a29-31c8-49d6-806a-6db0f80c9dbf)

PeshoOReda committed:
Feature/TA-7-Success-Page

PeshoOReda committed:
feat(success): create success screen

- Added a success image to assets.
- Added the success screen route.
- Created the success screen UI.

Signed-off-by: PeshoOReda <drmafiaemail13@gmail.com>

PeshoOReda committed:
refactor(success_screen): Update success screen layout and text

- Renamed `AppImages.successScreen` to `AppImages.successLogo` for clarity.
- Updated `AppRoutes` to use `successRoute` instead of `successScreen`.
- Modified the layout of the `SuccessScreen` to use `Center` for better alignment.
- Adjusted the size and fit of the success logo image.
- Updated text styles and alignment for "Thank You!!" and "The order delivered successfully" messages.
- Added line breaks to translation strings for better formatting in Arabic and English.

Signed-off-by: PeshoOReda <drmafiaemail13@gmail.com>

ahmedelazab1220 commented on lib/features/success_screen/presentation/view/success_screen.dart:
```diff
@@ -0,0 +1,54 @@
+    return Scaffold(
+      body: Center(
+        child: Padding(
+          padding: const EdgeInsets.all(30.0),
```
padding is 16 in design

ahmedelazab1220 commented on lib/features/success_screen/presentation/view/success_screen.dart:
```diff
@@ -0,0 +1,54 @@
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              Image.asset(
+                AppImages.successLogo,
+                fit: BoxFit.contain,
+                height: 150,
+                width: 150,
+              ),
```
make expanded is more better because fixed size may be crash on some screen!

PeshoOReda replied:
size of image is huuuuge Expanded not work correctly

ahmedelazab1220 reviewed (approved):
great work keep going 💯 👏

PeshoOReda committed:
refactor(success_screen): Adjust padding on success screen

- Changed the overall padding around the content on the success screen from 30.0 to 16.0.

Signed-off-by: PeshoOReda <drmafiaemail13@gmail.com>

Ahmed-Abdelghany-11 commented on lib/features/success_screen/presentation/view/success_screen.dart:
```diff
@@ -0,0 +1,54 @@
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              Image.asset(
```
making that is not wrong but use CachedNetworkImage is better for app performance

PeshoOReda replied:
this const from assets not from internet

Ahmed-Abdelghany-11 reviewed (approved):
great job ya peshoo ❤️
